src/model/resources/flexible_ccs/ccs_solvent_storage.jl

```diff
@@ -0,0 +1,389 @@
+                EP[:vOutput_CCS_SS][y, regenerator, t] * gen[y].steamuserate_mmbtu_per_ton)
+    # 3.3 Steam turbine capacity
+    @constraint(EP, cSteamTurbineCapacity[y in CCS_SOLVENT_STORAGE, t in 1:T], 
+                vOutput_CCS_SS[y, steamturbine, t] +
+                EP[:eSteam_regenerator][y ,t] / gen[y].heatrate_mmbtu_per_mwh_steamturbine_low <=
+                eTotalCap_CCS_SS[y, steamturbine])
+
+    # 4. Flow of power
+    # 4.1 Generated by the gas turbine: EP[:vOutput_CCS_SS][y, gasturbine, t]]
+    # 4.2 Generated by the steam turbine: EP[:vOutput_CCS_SS][y, steamturbine, t]]
+    @expression(EP, ePower_steamturbine_high[y in CCS_SOLVENT_STORAGE, t = 1:T],
+                EP[:eSteam_high][y, t] / gen[y].heatrate_mmbtu_per_mwh_steamturbine_high)
+    @expression(EP, ePower_steamturbine_mid[y in CCS_SOLVENT_STORAGE, t = 1:T],
+                EP[:eSteam_mid][y, t] / gen[y].heatrate_mmbtu_per_mwh_steamturbine_mid)
+    @expression(EP, ePower_steamturbine_low[y in CCS_SOLVENT_STORAGE, t = 1:T],
+                (EP[:eSteam_low][y, t] - EP[:eSteam_regenerator][y ,t]) / gen[y].heatrate_mmbtu_per_mwh_steamturbine_low)
+    @constraint(EP, cPower_steamturbine[y in CCS_SOLVENT_STORAGE, t = 1:T],
+                EP[:vOutput_CCS_SS][y, steamturbine, t] == EP[:ePower_steamturbine_high][y, t] + EP[:ePower_steamturbine_mid][y, t] + EP[:ePower_steamturbine_low][y, t])
+    # 4.3 Consumed by the absorber    
+    @expression(EP, ePower_absorber[y in CCS_SOLVENT_STORAGE, t = 1:T],
+                EP[:vOutput_CCS_SS][y, absorber, t] * gen[y].poweruserate_mwh_per_ton_co2_absorber)
+    # 4.4 Consumed by the compressor
+    @expression(EP, ePower_compressor[y in CCS_SOLVENT_STORAGE, t = 1:T],
+                EP[:vOutput_CCS_SS][y, compressor, t])
+    # 4.5 Consumed by other auxiliary loads
+    @expression(EP, ePower_other[y in CCS_SOLVENT_STORAGE, t = 1:T],
+                EP[:vFuel][y, t] * gen[y].poweruserate_mwh_per_mmbtu_fuel_other)
+    # 4.6 Blancen between the CCS_SOLVENT_STORAGE and the grid
+    @variable(EP, vCHARGE_CCS_SS[y in CCS_SOLVENT_STORAGE, t = 1:T] >= 0)  # Grid export to the system
+    @constraint(EP, cCharge[y in CCS_SOLVENT_STORAGE, t = 1:T],
+                EP[:ePower_absorber][y, t] + EP[:ePower_compressor][y, t] + EP[:ePower_other][y, t] >= vCHARGE_CCS_SS[y, t])
+    # link vP, vCHARGE_CCS_SS, and eP_CCS_SS
+    @expression(EP, eP_CCS_SS[y in CCS_SOLVENT_STORAGE, t=1:T], 
+                EP[:vOutput_CCS_SS][y, gasturbine, t] + EP[:vOutput_CCS_SS][y, steamturbine, t] - EP[:ePower_absorber][y, t] - 
+                EP[:ePower_compressor][y, t] - EP[:ePower_other][y, t] + vCHARGE_CCS_SS[y, t])
+    @constraint(EP, cP[y in CCS_SOLVENT_STORAGE, t = 1:T], eP_CCS_SS[y, t] == EP[:vP][y,t])
+    @expression(EP, ePowerBalance_CCS_SS[t = 1:T, z = 1:Z],
+                sum(EP[:eP_CCS_SS][y, t] - EP[:vCHARGE_CCS_SS][y, t]
+                for y in intersect(CCS_SOLVENT_STORAGE, resources_in_zone_by_rid(gen, z))))
```

Comment:
similar to the previous comment:
```suggestion
                for y in CCS_SOLVENT_STORAGE_BY_ZONE[z]))
```